### ISSUE TITLE
Pinned tabs 

### ISSUE THREAD

**Issue by JoaoNM** [minbrowser/min#1853]
Pinned tabs are a really useful feature and I think should be potentially quite simple to implement. Would be great to see this feature available!

**Comment by flightmansam** [comment]
What would you imagine the behaviour of a pinned tab would be? Present across all tasks? (and therefore) persistent across sessions?

**Comment by JoaoNM** [comment]
Yep exactly, like the one you might see in Safari: the tab icon is on the far left, and it's just the favicon of the website - makes for really clean & simple UI/UX I think
